app/Helpers/RemoteData.php

```diff
@@ -43,42 +45,47 @@ public static function sync()
         $users = collect($response->getBody()['value']);
 
         $result = array(
-            "total" => Pointer::count(),
             "successful" => 0,
             "failed"=> 0
         );
 
-        # TODO: get only pointers with category "Person"
-        Pointer::all()->each(function (Pointer $pointer) use ($users, &$result) {
+        foreach ($building->levels as $level) {
 
-            $filtered = $users->filter(function($value) use ($pointer){
+            foreach (Level::find($level->id)->pointers as $pointer) {
```

Comment:
why not directly iterate: 
```php
foreach($level->pointers as $pointer) {
   // process pointer
}
```

Reply:
in this case, we can save one db call, the pointers are eagerly loaded, so we can directly use them.

Reply:
agree, missed that for some reason